coldfront/core/project/management/commands/deactivate_ica_projects.py

```diff
@@ -0,0 +1,236 @@
+from decimal import Decimal
+
+from django.template.loader import render_to_string
+
+from coldfront.api.statistics.utils import get_accounting_allocation_objects, \
+    set_project_allocation_value, set_project_user_allocation_value
+from coldfront.config import settings
+from coldfront.core.allocation.models import AllocationAttributeType
+from coldfront.core.allocation.models import AllocationStatusChoice
+from coldfront.core.allocation.utils import get_project_compute_allocation
+from coldfront.core.project.models import Project
+from coldfront.core.project.models import ProjectStatusChoice
+from django.core.management.base import BaseCommand
+from django.db.models import Q
+import logging
+
+from coldfront.core.statistics.models import ProjectTransaction, \
+    ProjectUserTransaction
+from coldfront.core.utils.common import utc_now_offset_aware
+from coldfront.core.utils.mail import send_email_template
+
+"""An admin command that sets expired ICA Projects to 'Inactive' and
+their corresponding compute Allocations to 'Expired'."""
+
+
+class Command(BaseCommand):
+
+    help = (
+        'Set expired ICA Projects to \'Inactive\' and their corresponding '
+        'compute Allocations to \'Expired\'.')
+    logger = logging.getLogger(__name__)
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            '--dry_run',
+            action='store_true',
+            help='Display updates without performing them.')
+        parser.add_argument(
+            '--send_emails',
+            action='store_true',
+            default=False,
+            help='Send emails to PIs/managers about project deactivation.')
+
+    def handle(self, *args, **options):
+        """For each expired ICA Project, set its status to 'Inactive' and its
```

Comment:
It also sets the allocation's dates and possibly sends emails. We can go into less detail here or list everything.